hack/build-and-push.sh

```diff
@@ -40,7 +40,8 @@ VCS_REF=$(git rev-parse HEAD)
 declare -r ARTIFACT_TYPE_TEXT_XSHELLSCRIPT="text/x-shellscript"
 declare -r TEST_TASKS=${TEST_TASKS:-""}
 
-declare -r ANNOTATION_TASK_MIGRATION="dev.konflux-ci.task.migration"
+declare -r ANNOTATION_MIGRATION_EXISTS="dev.konflux-ci.task.migration-exists"
+declare -r ANNOTATION_IS_MIGRATION="dev.konflux-ci.task.is-migration"
```

Comment:
optional: would `has-migration` be more obvious than `migration-exists`?

Reply:
Yup, updated. This one should make Mykola happy as well. :smile: